### PR TITLE
add new message for triggering thoth-repo-init

### PR DESCRIPTION
**Pull request by KPostOffice**
## Related Issues and Dependencies

This will be consumed to schedule thoth-repo-init workflow

### CODE REVIEW

File: thoth/messaging/thoth_repo_init.py [hunk 1]
```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env python3
+# thoth-messaging
+# Copyright(C) 2020 Kevin Postlethwait
```
Comment (harshad16) [inline thoth/messaging/thoth_repo_init.py:3]:
```suggestion
# Copyright(C) 2021 Kevin Postlethwait
```

File: thoth/messaging/thoth_repo_init.py [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program. If not, see <http://www.gnu.org/licenses/>.
+
+
+"""This is Thoth Messaging module for MissingVersionMessage."""
```
Comment (harshad16) [inline thoth/messaging/thoth_repo_init.py:19]: is this description , right?

**Review by harshad16 — CHANGES_REQUESTED** [review]
lgtm
some doubts

**Review by fridex — APPROVED** [review]
lgtm 👍🏻 thanks!

**Pull request merged**